community/xorriso/depends

```diff
@@ -0,0 +1,3 @@
+bzip2
+musl
```

Comment:
Musl as a dependency doesn't need to be added as it will always exist. :)

Reply:
duh! thank you. fixed :)